.eslintrc

```diff
@@ -15,6 +15,9 @@ env:
 
 rules:
   no-console: 0
+  function-paren-newline: 0
+  object-curly-newline: 0
+  
```

Comment:
After updating `eslint` these two rules produced over 120 errors. I disabled them in the `.eslintrc` file